lib/LanguageServerNameImport/LanguageServerNameImportExtension.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace Phpactor\Extension\LanguageServerNameImport;
+
+use Phpactor\CodeTransform\Domain\Refactor\ImportName;
+use Phpactor\Container\Container;
+use Phpactor\Container\ContainerBuilder;
+use Phpactor\Container\Extension;
+use Phpactor\Extension\LanguageServer\LanguageServerExtension;
+use Phpactor\Extension\LanguageServerNameImport\Service\NameImport;
+use Phpactor\MapResolver\Resolver;
+
+class LanguageServerNameImportExtension implements Extension
```

Comment:
Instead of creating another extension, can this go in the `code-transform` extension?

Reply:
Yes. `code-transform` needs some changes anyway.

Reply:
With `code-transform" extension do you mean https://github.com/phpactor/language-server-phpactor-extensions/tree/master/lib/LanguageServerCodeTransform or https://github.com/phpactor/code-transform ?

Reply:
yeah, i mean the LanguageServerCodeTransform extension in this "package".

theoretically this package offers extensions which bridge the language server to phpactor's functionality. so `name import` is a `code-transform` thing, so should belong there.